src/wnaf.rs

```diff
@@ -94,12 +94,12 @@ pub(crate) fn wnaf_form<S: AsRef<[u8]>>(wnaf: &mut Vec<i64>, c: S, window: usize
     // Required by the NAF definition
     debug_assert!(window >= 2);
     // Required so that the NAF digits fit in i64
-    debug_assert!(window <= 64);
+    debug_assert!(window < 64);
```

Comment:
This seems like a separate issue, and fixing it in this PR is confusing. Rather than reducing the allowed values of `window`, we should fix the bug by making `width` a `u128`, and adjusting the rest of the code accordingly.